### PR TITLE
Client-side validation

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
Email in lower case only

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -189,3 +189,29 @@ document.querySelectorAll('.see-project').forEach((btn) => {
...
+// validation
+const form = document.querySelector('#contact form');
+const { email } = form.elements;
+
+function showMsg(input, msg) {
+  const small = document.createElement('small');
+  small.classList.add('error');
+  small.textContent = msg;
+  form.insertBefore(small, input);
+}
```
Comment (moise10r) [inline js/main.js:202]: - [ ] Good job so far :+1:  , however the show error message should be near the submit button as required, kindly fix it and request for another review

**Review by moise10r — CHANGES_REQUESTED** [review]
### **Hi @ismailakinkunmi ,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — APPROVED** [review]
## Hi, @ismailakinkunmi, @butlermuwo   👋

- [ ] [OPTIONAL] please in the future make sure to create a more descriptive pull request description. It's very important for people to know what your pull request introduces.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ismailakinkunmi** [comment]
adekunle ismail is inviting you to a scheduled Zoom meeting.

Topic: peer-code-issue
Time: Dec 24, 2021 10:30 PM West Central Africa

Join Zoom Meeting
https://microverse.zoom.us/j/93090102768?pwd=emRqVTBsc2t4cS9YREtWWm5QYk5BUT09

Meeting ID: 930 9010 2768
Passcode: 900757
One tap mobile
+13017158592,,93090102768#,,,,*900757# US (Washington DC)
+13126266799,,93090102768#,,,,*900757# US (Chicago)

Dial by your location
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 929 205 6099 US (New York)
        +1 253 215 8782 US (Tacoma)
        +1 346 248 7799 US (Houston)
        +1 669 900 6833 US (San Jose)
Meeting ID: 930 9010 2768
Passcode: 900757
Find your local number: https://microverse.zoom.us/u/adDRvUicPE

**Pull request merged**